cishouseholds/derive.py

```diff
@@ -34,7 +35,8 @@ def assign_survey_completed_status(
     df: DataFrame,
     column_name_to_assign: str,
     survey_completed_datetime_column: str,
-    survey_flushed_column: str,
+    survey_flushed_column: bool,
```

Comment:
Column value is bool but the parameter here should still be string for the column name